src/routes/+page.svelte

```diff
@@ -0,0 +1,9 @@
+<div>
+        <h1>Test</h1>
+        <p>This is pink</p>
+        <h2>I hope this works properly</h2>
+</div>
+
+<script>
+    import '../styles/global.css';
```

Comment:
use [`<link>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link) to include stylesheets. they are downloaded concurrently and cached better that way